python/example/ou_lif/lif.mod

```diff
@@ -0,0 +1,66 @@
+: Leaky-integrate and fire neuron
+: author Yannik Luboeinski
+: based on the LIF mechanism written by Sebastian Schmitt (https://github.com/tetzlab/FIPPA/blob/main/STDP/mechanisms/lif.mod)
```

Comment:
Would is also be OK to link here: https://github.com/arbor-contrib/FIPPA/blob/main/STDP/mechanisms/lif.mod ?
Just in case the Tetzlab changes their mind at any point ;)

Reply:
I guess so, @schmitts please comment if you object! Should the repo [arbor_ou_lif_example](https://github.com/jlubo/arbor_ou_lif_example) then also be forked by [arbor-contrib](https://github.com/arbor-contrib)?

Reply:
@brenthuisman is the expert on these kind of questions.